src/makeCalculator.js

```diff
@@ -4,7 +4,54 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  return {
+    get result() {
+      return result;
+    },
+
+    add(value) {
+      result += value;
+
+      return this;
+    },
+
+    subtract(value) {
+      result -= value;
+
+      return this;
+    },
+
+    multiply(value) {
+      result *= value;
+
+      return this;
+    },
+
+    divide(value) {
+      if (value === 0) {
+      } else {
+        result /= value;
+      }
+
+      return this;
```

Comment:
[CODE QUALITY] - The divide method should handle the case when `value` is 0. Currently, the `if` block is empty, which means dividing by zero will not change the result, but it would be clearer to throw an error or handle the case explicitly to avoid confusion.